src/components/tooltip/tooltip.tsx

```diff
@@ -114,7 +114,9 @@ export class Tooltip {
   }
 
   disconnectedCallback() {
-    this.popover.destroy();
+    if (this.popper) {
```

Comment:
Popper is exposed through the popover utility but doesn't exist in this context.